features/update/task_failure/feature.go

```diff
@@ -2,21 +2,29 @@ package task_failure
 
 import (
 	"context"
+	"strings"
 
+	enumspb "go.temporal.io/api/enums/v1"
 	"go.temporal.io/features/features/update/updateutil"
 	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/features/harness/go/history"
 	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/worker"
 	"go.temporal.io/sdk/workflow"
 )
 
 const (
-	execPanic      = "execPanic"
-	validatePanic  = "validatePanic"
-	shutdownSignal = "shutdown_signal"
+	panicMsg                = "fear not, dear examiner of logs, for this panic was intentional"
+	panicFromExecUpdate     = "panic_from_exec"
+	panicFromValidateUpdate = "panic_from_validate"
+	shutdownSignal          = "shutdown!"
 )
 
 var Feature = harness.Feature{
-	Workflows: Workflow,
+	Workflows:                          PanickyUpdates,
+	WorkerOptions:                      worker.Options{WorkflowPanicPolicy: worker.BlockWorkflow},
+	DisableWorkflowPanicPolicyOverride: true,
```

Comment:
:+1: I think (hope) this will be the only feature setting this anyways and I think is a great use of feature options (I expect the feature options list to grow and grow for these kinds of needs).